man/tss2-tcti-libtpms.7.in

```diff
@@ -0,0 +1,15 @@
+.\" Process this file with
+.\" groff -man -Tascii foo.1
+.\"
+.TH TCTI-LIBTPMS 7 "FEBRUARY 2021" "TPM2 Software Stack"
+.SH NAME
+tcti-libtpms \- TPM simulator library TCTI library
+.SH SYNOPSIS
+A TPM Command Transmission Interface (TCTI) module for interaction with the
+TPM2 simulator library libtpms.
+.SH DESCRIPTION
+tcti-libtpms is a library that abstracts the details of direct communication
+with the interface and protocol exposed by the daemon hosting the TPM2
+reference implementation. The interface exposed by this library is defined
+in the \*(lqTSS System Level API and TPM Command Transmission Interface
+Specification\*(rq specification.
```

Comment:
I thought libtpms was a library version of the backing implementation of a TPM. Does this still require a swtpm or something running?

I guess, thus far in my code review, I am missing the bigger picture of this.

Reply:
No, you're right. In fact, swtpm links against libtpms. I'll rewrite this to make it clear.